### PR TITLE
made login and findByEmail methods

### PR DESCRIPTION
**Pull request by Alex-Kharchuk**

### CODE REVIEW

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 1]
```diff
@@ -11,6 +15,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
```
Comment (KostyaNikiforov) [inline src/main/java/mate/academy/service/AuthenticationService.java:18]: Let's make this var field, by moving it to the class level. So you will not need to create a new UserService object each time this method is called.

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 2]
```diff
@@ -11,6 +15,13 @@ public class AuthenticationService {
...
+
+        if (userService.findByEmail(email) == null) {
+            return false;
+        }
+
+        User currentUser = userService.findByEmail(email);
```
Comment (KostyaNikiforov) [inline src/main/java/mate/academy/service/AuthenticationService.java:24]: Let's call the findByEmail method only once. And try to solve this task in one line

**Review by KostyaNikiforov — COMMENTED** [review]
Let's improve your solution)

File: src/main/java/mate/academy/service/AuthenticationService.java (hunk 2 above)
Comment (Alex-Kharchuk) [reply src/main/java/mate/academy/service/AuthenticationService.java:24]: Done in the new commit

**Pull request closed without merging**